dashboard/src/components/api/plugin-registry.factory.ts

```diff
@@ -71,8 +71,21 @@ export class PluginRegistry {
     return 'Che Editor';
   }
 
+  private getRegistryAuthorization(): string {
```

Comment:
Do you mind to have a service which will provide authorization header whenever it's necessary? Because now you have already got code duplication here and in `DevfileRegistry` class.

Reply:
fixed